alerts/thinpool-utilization.libsonnet

```diff
@@ -0,0 +1,63 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'thinpool-utilization',
+        rules: [
+          {
+            alert: 'GlusterThinpoolDataUtilization',
+            expr: |||
+                  100 * gluster_thinpool_data_used_bytes{%(glusterExporterSelector)s} / gluster_thinpool_data_total_bytes{%(glusterExporterSelector)s} > 80
+            ||| % $._config,
+            'for': $._config.thinpoolAlertTriggerTime,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Gluster Thinpool {{ $labels.thinpool }} Data Utilization more than 80%',
```

Comment:
Is there a label like this? I feel the label is `thinpool_name`. You should verify all the labels from https://github.com/gluster/gluster-prometheus/blob/master/docs/metrics.adoc

Reply:
changed.